src/test/java/thredds/server/metadata/util/NCMLModiferTest.java

```diff
@@ -0,0 +1,18 @@
+package thredds.server.metadata.util;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class NCMLModiferTest {
+
+    @Test
+    public void testFormat() throws Exception {
+        NCMLModifier ncmlModifier = new NCMLModifier();
+        double dbl = -1.6391277313232422E-6;
+        String fmtDbl = ncmlModifier.format(dbl);
+
+        assertEquals("-.0", fmtDbl);
```

Comment:
In decimal degrees, 5 decimal places is accurate to within +-1 meter, 6 decimal places accurate down to cm.  This pull request has it set to 5 decimal places, I'd recommend setting precision to 5-6 decimal places, beyond that and it implies an accuracy that does not likely exist.